src/vlrdevapi/commons/timezone.py

```
@@ -0,0 +1,212 @@
+    for zone_name in _TZ_ABBREV_MAP.values():
+        if zone_name in seen:
+            continue
+        seen.add(zone_name)
+        candidates.append(ZoneInfo(zone_name))
+
+    for zone in candidates:
+        local_naive = datetime.combine(local_date, local_time)
+        try:
+            localized = local_naive.replace(tzinfo=zone)
+        except Exception:
+            continue
```

Ruff (0.15.20)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)